usaspending_api/etl/management/commands/create_delta_table.py

```diff
@@ -34,7 +34,7 @@
     },
     "sam_recipient": {
         "schema_sql_string": sam_recipient_sql_string,
-        "source_table": "duns",
+        "source_table": "raw.duns",
```

Comment:
@collinwr did you find that you had to add this? Should be fine to not use this as long as you are using the API or ETL users.